data/publications.json

```diff
@@ -0,0 +1,7 @@
+[
+  {"id":10,"author":"Eshel Kedemi","title":"The effects of unique spatial characteristics of 3D-IVR and 2D computerized environments on cognitive modifiability.","year":2017,"citationCount":12},
```

Comment:
This seems like its just sample data and I don't know if it's hand generated but typically you would format the json object like so:
```
[
  {
    'id':10,
    'author':'Eshel Kedemi',
    'title':'The effects of unique spatial characteristics of 3D-IVR and 2D computerized environments on cognitive modifiability.',
    'year':2017,
    'citationCount':12
  },
]
```

Reply:
We should also use single quotes `'` instead of `"`